### PR TITLE
bsunanda:Run2-hcs23 Modify to add the adequate ESProducers in the configuration scenarios

### PR DESCRIPTION
**Pull request by bsunanda**
Modify many of the scenarios in COnfiguration/Geometry to add the adequate ESProducers

### CODE REVIEW

File: Configuration/Geometry/python/GeometrySLHCSimDB_cff.py [hunk 1]
```diff
@@ -7,3 +7,5 @@
 from GeometryReaders.XMLIdealGeometryESSource.cmsGeometryDB_cff import *
 from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
 from Geometry.TrackerNumberingBuilder.trackerNumbering2023GeometryDB_cfi import *
+from Geometry.HcalCommonData.hcalParameters_cfi      import *
```
Comment (ianna) [inline Configuration/Geometry/python/GeometrySLHCSimDB_cff.py]: @bsunanda - this is not needed - the parameters should come from DB

File: Configuration/Geometry/python/GeometrySimDB_cff.py [hunk 2]
```diff
@@ -7,3 +7,5 @@
 from GeometryReaders.XMLIdealGeometryESSource.cmsGeometryDB_cff import *
 from Geometry.MuonNumbering.muonNumberingInitialization_cfi import *
 from Geometry.TrackerNumberingBuilder.trackerNumberingGeometryDB_cfi import *
+from Geometry.HcalCommonData.hcalParameters_cfi      import *
```
Comment (ianna) [inline Configuration/Geometry/python/GeometrySimDB_cff.py]: @bsunanda - the same here, HcalParameters will be picked up from DB

Comment (bsunanda) [inline Configuration/Geometry/python/GeometrySimDB_cff.py]: I have updated this PR taking care of Yana's comments. Can this PR be tested and integrated soon?
________________________________

**Comment by ianna** [comment]
@cmsbuild please test

File: Configuration/Geometry/python/GeometryExtended2016Reco_cff.py [hunk 3]
```diff
@@ -30,4 +30,5 @@
 from Geometry.CaloEventSetup.EcalTrigTowerConstituents_cfi import *
 from Geometry.EcalMapping.EcalMapping_cfi import *
 from Geometry.EcalMapping.EcalMappingRecord_cfi import *
+from Geometry.HcalCommonData.hcalDDConstants_cff import *
```
Comment (ianna) [inline Configuration/Geometry/python/GeometryExtended2016Reco_cff.py]: @bsunanda - which PR pre seeds this one? I do not see this configuration fragment.

Comment (ianna) [inline Configuration/Geometry/python/GeometryExtended2016Reco_cff.py]: ok, it's in bsunanda/Run2-hcx22, but the definition in it is wrong for DB: hcalParameters_cfi should not be included in this case.

File: Configuration/Geometry/python/GeometryRecoDB_cff.py [hunk 4]
```diff
@@ -16,9 +16,9 @@
 from Geometry.CaloEventSetup.EcalTrigTowerConstituents_cfi import *
 from Geometry.EcalMapping.EcalMapping_cfi import *
 from Geometry.EcalMapping.EcalMappingRecord_cfi import *
+from Geometry.HcalCommonData.hcalDDConstants_cff import *
```
Comment (ianna) [inline Configuration/Geometry/python/GeometryRecoDB_cff.py]: @bsunanda - hcalParameters_cfi should not be included

Comment (bsunanda) [inline Configuration/Geometry/python/GeometryRecoDB_cff.py]: I think the hcalDDConstants_cff.py is already there in the IB of 2015-07-14-1100. I started from there.
________________________________

**Comment by ianna** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**